composer.json

```diff
@@ -15,17 +15,23 @@
     "require": {
         "php": "^8.0",
         "ext-bcmath": "*",
+        "ext-ctype": "*",
         "ext-exif": "*",
 		"ext-fileinfo": "*",
         "ext-gd": "*",
         "ext-json": "*",
+        "ext-mbstring": "*",
+        "ext-openssl": "*",
+        "ext-pdo": "*",
+        "ext-tokenizer": "*",
+        "ext-xml": "*",
+        "ext-zip": "*",
         "bepsvpt/secure-headers": "^7.1",
         "darkghosthunter/larapass": "dev-LycheeSpecial",
         "doctrine/dbal": "^3.1",
         "fideloper/proxy": "^4.3",
         "geocoder-php/cache-provider": "^4.3",
         "geocoder-php/nominatim-provider": "^5.5",
-        "graham-campbell/markdown": "^13.1",
```

Comment:
Why is this not needed anymore? We still use it, don't we?

Reply:
It was added [here](https://github.com/LycheeOrg/Lychee/commit/72c3d8b3ea1f574ea34b5611accd2f80c79ed88f)

Reply:
My fault. I did not see the line `use Illuminate\Mail\Markdown`. Then I am fine with that.

And yes, the `PageContent` is unused. Ildyria once wanted to add some "blog" functionality which would allow to create photo-blog pages with photos and text. But the feature has never been completed, i.e. you cannot add, edit nor delete pages. IMHO, this should have been move into some development branch until this feature does something actual useful, but ildyria wanted to keep it in the master branch.